gulpfile.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var gulp = require('gulp');
+var jshint = require('gulp-jshint');
+var gulpMocha = require('gulp-mocha');
+var appFiles = ['tcp_server.js'];
+var testFiles = ['./test/test.js'];
+
+gulp.task('jshint', function() {
+  return gulp.src(appFiles)
+    .pipe(jshint({
+      node: true,
+      globals: {
```

Comment:
remember, you can always put these globals in a .jshintrc file in your project. And then from there you can reuse that file in future projects.